src/app/action-playground/action-playground.component.html

```diff
@@ -87,3 +87,6 @@
     <p style="display: inline-block; padding-left: 2%"> {{confirmationDialogResponse}} </p>
 </div>
 
+<p>GND Directive (Integrated Authority File)</p>
```

Comment:
Please add an `<hr>` here :)

Reply:
done in https://github.com/dasch-swiss/dsp-ui-lib/pull/162/commits/1a9266c3a439a124f73b689c83f30443ede2b462